cmd/registry/cmd/vocabulary/unique.go

```diff
@@ -60,6 +61,6 @@ func uniqueCommand(ctx context.Context) *cobra.Command {
 		},
 	}
 
-	cmd.Flags().StringVar(&outputID, "output_id", "vocabulary-unique", "Artifact ID to use when saving each result vocabulary")
+	cmd.Flags().String("output_id", "vocabulary-unique", "Artifact ID to use when saving each result vocabulary")
```

Comment:
Was there an issue with this one? It's working for me.

Reply:
I accidentally changed this when I was checking all of the "output" related flags. I'll revert this one, but want to revisit the consistency question (below).